tests/connection/connection.cpp

```diff
@@ -0,0 +1,546 @@
+/*
+Copyright 2015 University of Auckland
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.Some license of other
+*/
+
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+TEST(Variable, addAndGetEquivalentVariable) {
+    libcellml::VariablePtr v1 = std::make_shared<libcellml::Variable>();
+    libcellml::VariablePtr v2 = std::make_shared<libcellml::Variable>();
+    libcellml::Variable::addEquivalence(v1,v2);
+    EXPECT_EQ(v2,v1->getEquivalentVariable(0));
```

Comment:
Missing space before ```v1``` (same for ```v2``` at lines 24, 31 and 32). (Same elsewhere in the code.)